frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.scss

```diff
@@ -1,6 +1,10 @@
 .odc-monitoring-dashboard-graph {
+  margin-bottom: var(--pf-global--gutter--md);
+  &:last-of-type {
+    margin-bottom: 0;
+  }
   .query-browser__wrapper {
     border: none;
-    margin: 0;
+    padding: 0;
   }
```

Comment:
```
.query-browser__wrapper {
    border: 1px solid #ccc;
    margin: 0 0 20px 0;
    overflow: visible;
    padding: 10px;
    width: 100%;
}
```

Not sure we can completely kill the padding and just accept margin. The chart gets pretty close to the header. 

With no alterations (taking the padding & margin from above):
![Screen Shot 2020-03-03 at 1 20 48 PM](https://user-images.githubusercontent.com/8126518/75806611-ff965d80-5d51-11ea-957d-bf8ef257fefe.png)

And then without padding (but keeping margin):
![Screen Shot 2020-03-03 at 1 21 01 PM](https://user-images.githubusercontent.com/8126518/75806634-0cb34c80-5d52-11ea-9576-9235c803df6b.png)

Margin isn't really helping a lot, there is already whitespace under the chart, but padding looks to be quite useful... Was this just a subjective change @jeff-phillips-18?

Reply:
Leaving the padding to 10px gives unnecessary padding around the chart. The DashboardBody component should control the spacing. Leaving the 10px padding around the chart gives this (padding in green):

![image](https://user-images.githubusercontent.com/11633780/75809347-a7ae2580-5d56-11ea-98da-60b05e0d99b6.png)

which I don't believe is desireable.

Reply:
Perhaps the spacing left & right is wasteful... I think the padding top has value to keep a spacer between the graph labels & the header spacer line. The bottom spacing is as useful as the margin I suspect.

Thoughts @jeff-phillips-18 ?

Reply:
The margin used is for the bottom legend labels and I believe ought not be changed.
The spacing between the labels and the bottom of the card is part of the card styling and should be left to be consistent with other cards.

I agree there could be more top spacing. I'll look to make it consistent with other cards.

Reply:
![image](https://user-images.githubusercontent.com/11633780/75813758-8d784580-5d5e-11ea-8804-ec7905d06fa2.png)

Reply:
Looking pretty good, thanks! I'll check it out in a moment to do one last pass before 👍